drivers/bridge/bridge.go

```diff
@@ -402,6 +395,14 @@ func (d *driver) configure(option map[string]interface{}) error {
 		iptables.OnReloaded(func() { logrus.Debugf("Recreating iptables chains on firewall reload"); setupIPChains(config) })
 	}
 
+	if config.EnableIPForwarding {
+		err = setupIPForwarding()
+		if err != nil {
+			logrus.Warn(err)
+			return err
+		}
+	}
+
```

Comment:
We will now touch the iptables (changing the default policy) even if daemon was started with `--iptables=false`.
I guess it is fine, given we are addressing a vulnerability.
We'll need to document this behavior.